pkg/agent/util/ethtool.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
```

Comment:
It is said util is not a recommended way name a golang package. Would you consider another meaning full name?

Reply:
This package pkg/agent/util/ is already there, actually I created it when looking for a place to put a real util function specific to agent. My opinion is as long as there is a scope for the util, like "foo/bar/util" or "foor/util/bar", it doesn't conflict with the recommendation, otherwise we may end up with having a package for every piece of util function. What do you think? BTW, Kubernetes and istio code doesn't escape from this problem too but of course these util packages are scoped to a specific module.

Reply:
Feel ethtool is quite different from GenerateContainerInterfaceName() to be put into the same util package. Should we create a package like linuxnet and put iptables and ethtool there?

Reply:
Based on examples from K8s (https://github.com/kubernetes/kubernetes/tree/master/pkg/util/iptables), it seems that `util/ethtool/ethtool_linux.go` could be a good choice. I'm going to make that change.